src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -13,19 +14,24 @@ public int formula(int age, int restingHR, double intensity) {
         return bpm;
     }
 
+    //Naming / data-type?
     public String[] checkDifferentIntensities(int age, int restingHR) {
+        //Shorter way to do addition?
         for (double intensity = 55; intensity <= 95; intensity = intensity + 5) {
+            //Brackets and whitespace?
             table[index] = (formula(age, restingHR, intensity) + "");
 
+            //Why the additional counter?
             if(index < table.length){
```

Comment:
Not really sure why we have this additional check on the index bounds here, the condition in our outer-loop should be enough.  Also could you rephrase the loop so there's only one counter rather than two?